Gemfile.lock

```diff
@@ -519,3 +521,6 @@ DEPENDENCIES
   uglifier (~> 4.2.0)
   web-console (~> 3.7)
   webmock (~> 3.11)
+
```

Comment:
We don't fix the version of Bundler intentionally so it can be deployed on any environment regardless of the version of bundler they may have

Reply:
Alright, I've updated Gemfile.lock